docs/advanced-topics.asciidoc

```diff
@@ -3,8 +3,35 @@
 
 * <<custom-transactions, Custom Transactions>>
 * <<custom-transaction-name, Initial page load transaction names>>
+* <<longtasks>>
 * <<typescript>>
 
+[[longtasks]]
+=== How to interpret Longtask Spans in the UI
+
+Longtasks is a new performance metric that helps measure the responsiveness and
+helps understand the bad user experience on the web. It enables detecting tasks
```

Comment:
```suggestion
helps application developers to understand the user experience on their application. It enables detecting tasks
```

Reply:
changed this a bit since application was called in multiple places.